less/admin/AdminNav.less

```diff
@@ -1,17 +1,92 @@
-@admin-pane-width: 300px;
+@admin-pane-width: 250px;
 
 .App {
   padding-bottom: 0;
 }
-.AdminLinkButton-description {
-  display: none;
+.App-content {
```

Comment:
This doesn't seem to have a significant effect. Is it needed? (the addition I mean, maybe remove it entirely?)